### ISSUE TITLE
StripeObject.last_response is None on save() calls

### ISSUE THREAD

**Issue by jleclanche** [stripe/stripe-python#393]
Repro steps from #371:

```py
customer = stripe_customer.api_retrieve()
customer.account_balance -= 250
customer.save(idempotency_key=ik)
credit_request_id = customer.last_response.request_id  # Breaks! last_response is None
```

**Comment by ob-stripe** [comment]
Thanks @jleclanche. I was able to reproduce this easily, will try to push a fix soon.

**Comment by ob-stripe** [comment]
Fixed in v1.77.2.
